### PR TITLE
fix(DB/creature): High Admiral "Shelly" Jorrik Rep and spawn

### PR DESCRIPTION
**Pull request by BeefSteaker**
## Changes Proposed:
-  Shelly gives 5 bloodsail rep and -25 Booty bay (+spillover) Rep
-  Set respawn time to 8 hours

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/13200
- Closes https://github.com/chromiecraft/chromiecraft/issues/4149
- Partially fixes https://github.com/azerothcore/azerothcore-wotlk/issues/13201

## Tests Performed:

- killed NPC on local
- 

## How to Test the Changes:

.go c 1975960
at war with booty bay
/tar high
.damage 4000

## Known Issues and TODO List:

- [ ] Spawn time set to 8 hours, sources indicate that this is the minimum, and it is somewhere between 8 and 48 hours
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/repandspawn.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+--lots of Reputation changes
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/repandspawn.sql:1]:
```suggestion
-- lots of Reputation changes
```

File: data/sql/updates/pending_db_world/repandspawn.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepFaction1`=87, `RewOnKillRepFaction2`=21, `MaxStanding1`=5, `IsTeamAward1`=0, `RewOnKillRepValue1`=5, `MaxStanding2`=7, `IsTeamAward2`=1, `RewOnKillRepValue2`=-25, `TeamDependent`=0 WHERE `creature_id`=26081;
+--Spawn to 8 hours
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/repandspawn.sql:3]:
```suggestion
-- Spawn to 8 hours
```

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/repandspawn.sql [hunk 3]
```diff
@@ -0,0 +1,4 @@
+--lots of Reputation changes
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepFaction1`=87, `RewOnKillRepFaction2`=21, `MaxStanding1`=5, `IsTeamAward1`=0, `RewOnKillRepValue1`=5, `MaxStanding2`=7, `IsTeamAward2`=1, `RewOnKillRepValue2`=-25, `TeamDependent`=0 WHERE `creature_id`=26081;
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/repandspawn.sql:2]:
```suggestion
DELETE FROM `creature_onkill_reputation` WHERE `creature_id`=26081;
INSERT INTO `creature_onkill_reputation` (`creature_id`, `RewOnKillRepFaction1`, `RewOnKillRepFaction2`, `MaxStanding1`, `IsTeamAward1`, `RewOnKillRepValue1`, `MaxStanding2`, `IsTeamAward2`, `RewOnKillRepValue2`, `TeamDependent`) VALUES (26081, 87, 21, 5, 0, 5, 7, 1, -25, 0);
```

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/repandspawn.sql (hunk 3 above)
Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/repandspawn.sql:2]: just fyi. 26081 entry does not exsist currently in the db,  for that creature_onkill_reputation table.
updating something that does not exsist will lead to a error.
the above is my proposed delete and insert query.

File: data/sql/updates/pending_db_world/repandspawn.sql (hunk 1 above)
Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/repandspawn.sql:1]: now the purpose of spacing after -- is for the ci runs. --XX and -- XX is viewed differently with the workflow.

File: data/sql/updates/pending_db_world/repandspawn.sql (hunk 2 above)
Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/repandspawn.sql:3]: now the purpose of spacing after -- is for the ci runs. --XX and -- XX is viewed differently with the workflow.

**Review by acidmanifesto — APPROVED** [review]

File: data/sql/updates/pending_db_world/repandspawn.sql (hunk 3 above)
Comment (BeefSteaker) [reply data/sql/updates/pending_db_world/repandspawn.sql]: Thanks for the corrections

**Comment by acidmanifesto** [comment]
Thank you for the PR

**Pull request merged**